docs/getting_started.rst

```diff
@@ -56,3 +61,133 @@ is a more complete example that accepts alerts from Alertmanager:
 This example sets up a webhook to receive events from alertmanager and then matches events
 where the `fastapi` job alert has a status of `firing`.  This runs a playbook that will
 remediate the issue.
+
+
+Let's build an example rulebook that will trigger an action from a
+webhook. We will be looking for a specific payload from the webhook, and
+if that condition is met from the webhook event, then ansible-rulebook
+with trigger the desired action. Below is our example rulebook:
+
+.. code-block:: yaml
+
+   ---
+   - name: Listen for events on a webhook
+     hosts: all
+
+     ## Define our source for events
+
+     sources:
+       - ansible.eda.webhook:
+           host: 0.0.0.0
+           port: 5000
+
+     ## Define the conditions we are looking for
+
+     rules:
+       - name: Say Hello
+         condition: event.payload.message == "Ansible is super cool"
+
+     ## Define the action we should take should the condition be met
+
+         action:
+           run_playbook:
+             name: say-what.yml
+
+The playbook ``say-what.yml``:
+
+.. code-block:: yaml
+
+   - hosts: localhost
+     connection: local
+     tasks:
+       - debug:
+           msg: "Thank you, my friend!"
+
+If we look at this example, we can see the structure of the rulebook.
+Our sources, rules and actions are defined. We are using the webhook
+source plugin from our ansible.eda collection, and we are looking for a
+message payload from our webhook that contains “Ansible is super cool”.
+Once this condition has been met, our defined action will trigger which
+in this case is to trigger a playbook.
+
+One important thing to take note of ansible-rulebook is that it is not
+like ansible-playbook which runs a playbook and once the playbook has
+been completed it will exit. With ansible-rulebook, it will continue to
+run waiting for events and matching those events, it will only exit upon
+a shutdown action or if there is an issue with the event source itself,
+for example if a website you are watching with the url-check plugin
+stops working.
```

Comment:
This is not entirely correct. The CLI will also exit when the condition in a ruleset has been met and the action has been run.